hipaa-password-input.html

```diff
@@ -3,8 +3,8 @@
 <link rel="import" href="../paper-input/paper-input-behavior.html">
```

Comment:
@sricharan123 
I don't see the paper-input-behavior used explicitly anywhere in your code.
I think this line should be removed.